.github/workflows/build.yml

```diff
@@ -90,6 +90,35 @@ jobs:
           make ci
           make sanity 
 
+  ansible:
+    runs-on: ubuntu-latest
+    needs: ['lint', 'unit-test', 'bdd-test']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set Image Org
+        # sets the default IMAGE_ORG to openebs
+        run: |
+          [ -z "${{ secrets.IMAGE_ORG }}" ] && IMAGE_ORG=openebs || IMAGE_ORG=${{ secrets.IMAGE_ORG}}
+          echo "IMAGE_ORG=${IMAGE_ORG}" >> $GITHUB_ENV
+
```

Comment:
Add the buildx install action
```yaml
-  name: Set up Docker Buildx
   uses: docker/setup-buildx-action@v1
```

Reply:
We are not using build and pushing multi arch images as of now, so will not be need to add this